src/routes/loginOAuth.js

```diff
@@ -40,4 +40,43 @@ server.post("/auth/login", async (req, res) => {
   }
 });
 
+server.post("/auth/changePassword", async (req, res) => {
+  const { username, oldPassword, newPassword } = req.body;
+  let encryptedPassword
+  await encrypt.encryptPassword(newPassword).then(encryptedPass => {
+    encryptedPassword = encryptedPass;
+  })
+
+  const user = await Users.findOne({ username });
+  if (!user) {
+    res.status(401).json({
+      message: "Username not found",
+      error: "401: User not found",
+    });
+  } else {
```

Comment:
We don't need this else statement anymore.
```suggestion
  }
```